streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorContextUtils.java

```diff
@@ -39,9 +39,7 @@ private ProcessorContextUtils() {}
      * removing the need for this method.
      */
     public static long getCurrentSystemTime(final ProcessorContext context) {
```

Comment:
nit (not introduced in this PR, but would be a nice side cleanup): rename to `currentSystemTime(...)`